.github/workflows/ci-lai-test-full.yml

```diff
@@ -0,0 +1,235 @@
+name: Test LAI full
```

Comment:
What's the benefit of having a monolithic workflow to run everything?

This makes things unnecessarily complicated, the filters need to account for everything, there's skip and conditions everywhere

Reply:
I think since we're doing mangling of paths in creating LAI package, we need to be sure, everything works as expected.

Reply:
@justusschock mentioned that we somewhere override sys path so we need to be sure that it will be working even from the created mono-package

Reply:
I don't understand how that relates to merging all testing workflows

Reply:
@carmocca it does not, we will split them after we are successfully running them!